ts/routes/graphs/reviews.ts

```diff
@@ -340,7 +340,7 @@ export function renderReviews(
         averageAnswerTime: string,
         averageAnswerTimeLabel: string;
     if (showTime) {
-        totalString = timeSpan(total / 1000, false);
+        totalString = timeSpan(total / 1000, false, true, TimespanUnit.Hours);
```

Comment:
IMO, we can set precise to False here because, for example, showing 52.26 hours of study time doesn't provide much additional information as compared to showing 52 hours of study time.

```suggestion
        totalString = timeSpan(total / 1000, false, false, TimespanUnit.Hours);
```

Reply:
I understand your point. Perhaps, we should skip this for now and wait for a subsequent overhaul of the rounding system (https://forums.ankiweb.net/t/reduce-precision-of-time-studied-today/42309).